atlasdb-workload-server-api/build.gradle

```diff
@@ -4,7 +4,11 @@ dependencies {
     implementation 'com.fasterxml.jackson.core:jackson-annotations'
     implementation 'com.fasterxml.jackson.core:jackson-core'
     implementation 'com.fasterxml.jackson.core:jackson-databind'
+    implementation 'com.palantir.safe-logging:preconditions'
     implementation 'com.google.guava:guava'
+    implementation 'io.vavr:vavr'
```

Comment:
We expose the vavr `Map` in the public-API `ValidationStore`, so this should be an `api` dependency: otherwise, a user importing this project/jar will get weird compile breaks if they don't already pull in vavr.